### PR TITLE
Recompilation of assets.

### LINKED ISSUE #31: App is throwing an exception on creation of a new user:
```
2016-02-25T20:06:40.805595+00:00 app[web.1]:   Rendered user_mailer/signup_email.html.erb (5.1ms)
2016-02-25T20:06:40.808610+00:00 app[web.1]: 
2016-02-25T20:06:40.808612+00:00 app[web.1]: ActionView::Template::Error (fb.png isn't precompiled):
2016-02-25T20:06:40.808613+00:00 app[web.1]:     623:                                                         <% twitter_message = "Check out The Mindfulness Edge book!" %>
2016-02-25T20:06:40.808614+00:00 app[web.1]:     624:   
2016-02-25T20:06:40.808615+00:00 app[web.1]:     625:                                                         <div class="social-links">
2016-02-25T20:06:40.808617+00:00 app[web.1]:     626:                                                           <a href="http://www.facebook.com/sharer/sharer.php?u=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&title=The Mindfulness Edge" class="fb" target="_blank"><%= image_tag("fb.png", class: 'fb_img') %></a>
2016-02-25T20:06:40.808618+00:00 app[web.1]:     627:                                                           <div class="sep"></div>
2016-02-25T20:06:40.808619+00:00 app[web.1]:     628:                                                           <a href="http://twitter.com/share?url=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&text=<%= CGI::escape(twitter_message) %>" class="twit" target="_blank"><%= image_tag("twit.png", class: 'tweet_img') %></a>
2016-02-25T20:06:40.808620+00:00 app[web.1]:     629:                                                         </div>
2016-02-25T20:06:40.808621+00:00 app[web.1]:   app/views/user_mailer/signup_email.html.erb:626:in `_app_views_user_mailer_signup_email_html_erb__1889170839201358767_70356104563160'
2016-02-25T20:06:40.808621+00:00 app[web.1]:   app/mailers/user_mailer.rb:7:in `signup_email'
2016-02-25T20:06:40.808624+00:00 app[web.1]: 
2016-02-25T20:06:40.808622+00:00 app[web.1]:   app/models/user.rb:58:in `send_welcome_email'
2016-02-25T20:06:40.808623+00:00 app[web.1]:   app/controllers/users_controller.rb:34:in `create'
2016-02-25T20:06:40.808624+00:00 app[web.1]: 
2016-02-25T20:06:40.806926+00:00 app[web.1]: Completed 500 Internal Server Error in 1141ms
peaty:prelaunchr_for_ni jchapin$ 

```

### PR DESCRIPTION
**Pull request by jchapin**
Fixes #31

### CODE REVIEW

**Comment by jchapin** [comment]
Heroku appears to be offline for pushing for the moment.

**Comment by jchapin** [comment]
https://status.heroku.com

**Comment by jchapin** [comment]
https://status.heroku.com/incidents/860

**Comment by jchapin** [comment]
```
remote: !	Heroku API is temporarily unavailable.
remote: !	For more information, visit: https://status.heroku.com

remote: !	Heroku has temporarily disabled this feature, please try again shortly.
remote: !	See http://status.heroku.com for current Heroku platform status.
```

**Comment by RanSolo** [comment]
:+1: I was wondering what was going on

**Comment by jchapin** [comment]
```
2016-02-25T22:15:42.569286+00:00 heroku[router]: at=info method=POST path="/users" host=ni-prelaunchr-dev.herokuapp.com request_id=1ab1b6a6-cb9a-49c4-bf55-a3afb19d2033 fwd="216.127.112.163" dyno=web.1 connect=1ms service=721ms status=500 bytes=903
2016-02-25T22:15:42.546642+00:00 app[web.1]:   Rendered user_mailer/signup_email.html.erb (6.5ms)
2016-02-25T22:15:42.556698+00:00 app[web.1]: 
2016-02-25T22:15:42.556701+00:00 app[web.1]: ActionView::Template::Error (fb.png isn't precompiled):
2016-02-25T22:15:42.556703+00:00 app[web.1]:     623:                                                         <% twitter_message = "Check out The Mindfulness Edge book!" %>
2016-02-25T22:15:42.556703+00:00 app[web.1]:     624:   
2016-02-25T22:15:42.556704+00:00 app[web.1]:     625:                                                         <div class="social-links">
2016-02-25T22:15:42.556706+00:00 app[web.1]:     626:                                                           <a href="http://www.facebook.com/sharer/sharer.php?u=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&title=The Mindfulness Edge" class="fb" target="_blank"><%= image_tag("fb.png", class: 'fb_img') %></a>
2016-02-25T22:15:42.556706+00:00 app[web.1]:     627:                                                           <div class="sep"></div>
2016-02-25T22:15:42.556707+00:00 app[web.1]:     628:                                                           <a href="http://twitter.com/share?url=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&text=<%= CGI::escape(twitter_message) %>" class="twit" target="_blank"><%= image_tag("twit.png", class: 'tweet_img') %></a>
2016-02-25T22:15:42.556723+00:00 app[web.1]:   app/controllers/users_controller.rb:34:in `create'
2016-02-25T22:15:42.556708+00:00 app[web.1]:     629:                                                         </div>
2016-02-25T22:15:42.556722+00:00 app[web.1]:   app/models/user.rb:58:in `send_welcome_email'
2016-02-25T22:15:42.556720+00:00 app[web.1]:   app/views/user_mailer/signup_email.html.erb:626:in `_app_views_user_mailer_signup_email_html_erb__2817537981676106998_69929922919100'
2016-02-25T22:15:42.556721+00:00 app[web.1]:   app/mailers/user_mailer.rb:7:in `signup_email'
2016-02-25T22:15:42.556723+00:00 app[web.1]: 
2016-02-25T22:15:42.556724+00:00 app[web.1]: 
peaty:prelaunchr_for_ni jchapin$ 
```

precompiling issue still remains.

File: config/environments/production.rb [hunk 1]
```diff
@@ -50,6 +50,7 @@
   # config.assets.precompile += %w( search.js )
   config.assets.precompile += %w( core.css bootstrap.css reset.css )
   config.assets.precompile += %w( ie.css )
+  config.assets.precompile += %w[ *.png *.jpeg *.jpg *.gif ]
```
Comment (jchapin) [inline config/environments/production.rb]: Critical line 1

File: config/environments/production.rb [hunk 2]
```diff
@@ -89,6 +90,8 @@
   config.action_mailer.default_url_options = {
     host: 'www.themindfulnessedge.com'
   }
+  config.action_controller.asset_host = "https://#{ENV['APP_HOSTNAME']}"
+  config.action_mailer.asset_host = config.action_controller.asset_host
```
Comment (jchapin) [inline config/environments/production.rb]: Critical lines 2 & 3

**Pull request merged**